_layouts/default.html

```diff
@@ -12,6 +12,7 @@
         {%- include sidebar.html -%}
       </aside>
       <div class="page-content">
+        <a href="{{ site.github.repository_url }}/tree/master/{{ page.path }}">{{ site.data.common[page.lang].button.edit }}</a>
```

Comment:
The URL should be `{{ site.github.repository_url }}/edit/master/{{ page.path }}`, so that it opens GitHub editor directly 😉